optional/capi/constants_spec.rb

```diff
@@ -39,8 +41,10 @@
     @s.rb_cFile.should == File
   end
 
-  specify "rb_cFixnum references the Fixnum class" do
-    @s.rb_cFixnum.should == Fixnum
+  if CApiConstantsSpecs.method_defined?(:rb_cFixnum)
```

Comment:
These should be Ruby version ranges so they also inform when is the transition. I'll fix it.